sonic_eeprom/eeprom_base.py

```diff
@@ -13,6 +13,9 @@
 
 try:
     import exceptions
+except ImportError:
+    import builtins as exceptions
```

Comment:
Suggest to add comments here for posterity's sake:

```
try:
    import exceptions              # Python 2
except ImportError:
    import builtins as exceptions  # Python 3
```

Reply:
Added comments for all the modifications behind both python 2 and python 3 support